etcd3/locks.py

```diff
@@ -1,8 +1,8 @@
+import threading
+import time
 import uuid
 
-import tenacity
```

Comment:
The dependency is removed completely for installing the package.
It is only used in tests.